server/datastore/mysql/migrations/tables/20231025160156_AddHostUuidColumnToMdmWindowsEnrollments.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231025160156, Down_20231025160156)
+}
+
+func Up_20231025160156(tx *sql.Tx) error {
+	stmt := `
+          ALTER TABLE mdm_windows_enrollments
+          ADD COLUMN host_uuid varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT ''
```

Comment:
I imagine that we'll want to add some indexing around this but will leave that for the future.